cassandra/go.mod

```diff
@@ -0,0 +1,12 @@
+module github.com/gofiber/storage/cassandra
+
+go 1.18
```

Comment:
This should be go 1.19

Reply:
sure. do you need this to be at least 1.19 or exact 1.19 ? @gaby